internal/core/platform.rs

```diff
@@ -98,18 +98,19 @@ pub trait Platform {
 }
 
 /// The clip board, used in [`Platform::clipboard_text`] and [Platform::set_clipboard_text`]
+#[repr(u8)]
```

Comment:
You could also use `repr(C)` and then use the enum directly in the C function signature.

```suggestion
#[repr(C)]
```

Reply:
if this is an `#[repr(u8)]` it is fine if it is an enum class that use uint8_t as a underlying type